tracee-rules/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/aquasecurity/tracee/tracee-rules/engine"
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:  "tracee-rules",
+		Usage: "A rule engine for Runtime Security",
+		Action: func(c *cli.Context) error {
+			sigs, err := getSignatures(c.String("rules-dir"), c.StringSlice("rules"))
+			if err != nil {
+				return err
+			}
+			var inputTracee chan types.Event
+			if c.IsSet("tracee-file") {
+				inputTracee, err = setupTraceeSource(c.String("tracee-file"))
+			} else {
+				inputTracee, err = setupStdinSource()
+			}
+			if err != nil || inputTracee == nil {
+				return err
+			}
+			output, err := setupOuput(c.String("webhook"))
+			if err != nil {
+				return err
+			}
+			e := engine.NewEngine(sigs, inputTracee, output)
+			e.Start(sigHandler())
+			return nil
+		},
+		Flags: []cli.Flag{
+			&cli.StringSliceFlag{
+				Name:  "rules",
+				Usage: "select which rules to load",
+			},
+			&cli.StringFlag{
+				Name:  "rules-dir",
+				Usage: "directory where to search for rules",
+			},
+			&cli.StringFlag{
+				Name:  "webhook",
+				Usage: "call this HTTP endpoint for every match",
+			},
+			&cli.StringFlag{
+				Name:  "tracee-file",
+				Usage: "path to Tracee Gob output file",
+			},
```

Comment:
when the sigs were compiled in, --list was necessary to tell the users which sigs were compiled in. now that sigs are not compiled, but loaded from a local directory, it's no longer necessary. I can still add it, just as a validation. 

for input, we currently support just tracee, which is given by the --tracee-file flag. since the user has to provide the input coordinates anyway, I don't see the need for another flag that says which inputs are used. if you give an input coordinates you tell us to use it

Reply:
Other than printing the signature names, we also printed their metadata, which includes the tags and description.
I think it is usefull for the user to have this functionality, otherwise he will need to open the signature files to get this data

Reply:
added --list